### ISSUE TITLE
Detecting when Camera Card is inserted?

### ISSUE THREAD

**Issue by latenitefilms** [Hammerspoon/hammerspoon#1051]
An annoying feature of Final Cut Pro is that whenever you insert an camera card (such as an SD or CF card from a stills camera) with Final Cut Pro open, it will launch a "Media Import" window - basically, the same functionality as Apple's Photos app - but with no "Open Photos for this device" option, so you can't turn it off.

Does anyone know how Apple's detecting the insertion of the camera card? Is there anyway to do the same thing in Hammerspoon?

Basically, what I'd like to do is detect whenever a camera card is inserted, then if the "Media Import" window opens in Final Cut Pro, I'll close it instantly.

Any ideas @asmagill ?

**Comment by latenitefilms** [comment]
Solved with **hs.fs.volume**.
